pac4j-core/src/main/java/org/pac4j/core/engine/DefaultCallbackLogic.java

```diff
@@ -93,10 +105,10 @@ public R perform(final C context, final Config config, final HttpActionAdapter<R
     }
 
     protected void saveUserProfile(final C context, final Config config, final CommonProfile profile,
-                                   final boolean multiProfile, final boolean renewSession) {
+                                   final boolean saveInSession, final boolean multiProfile, final boolean renewSession) {
         final ProfileManager manager = getProfileManager(context, config);
         if (profile != null) {
-            manager.save(true, profile, multiProfile);
+            manager.save(saveInSession, profile, multiProfile);
```

Comment:
Same comment here: can we directly use the `saveInSession` property and no input parameter?

Reply:
OK